src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 <div>
-  <AddressAutocomplete apiKey="YOUR_API_KEY_HERE" :addresses="addresses" @selectItem="selectItem" @newSuggestions="addNewSuggestions" />
+  <AddressAutocomplete apiKey="live_pub_16d30adbb46a1c360ebf7d1e6b48361" :addresses="addresses" @selectItem="selectItem" @newSuggestions="addNewSuggestions" />
```

Comment:
small nit but replace this with your api key here

Reply:
nice catch! 🙏